docs/feature_rgb_matrix.md

```diff
@@ -562,7 +562,9 @@ static bool my_cool_effect2(effect_params_t* params) {
 #endif // RGB_MATRIX_CUSTOM_EFFECT_IMPLS
 ```
 
-For inspiration and examples, check out the built-in effects under `quantum/rgb_matrix_animations/`
+For inspiration and examples, check out the built-in effects under [`quantum/rgb_matrix/animations/`][rgb-matrix-animations-dir].
+
+[rgb-matrix-animations-dir]: https://github.com/qmk/qmk_firmware/tree/master/quantum/rgb_matrix
```

Comment:
```suggestion
For inspiration and examples, check out the built-in effects under `quantum/rgb_matrix/animations/`.
```
Should also be done for LED Matrix if necessary.

Reply:
Thanks, I guess you don't want to link to github.com?